bee-common/bee-common-derive/tests/packable/fail/duplicated_tag_enum.rs

```diff
@@ -0,0 +1,20 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+#![allow(unused_imports)]
+
+use bee_common::packable::Packable;
+
+use core::convert::Infallible;
+
+#[derive(Packable)]
+#[packable(tag_ty = u8)]
+#[packable(error = Infallible)]
```

Comment:
hmmm... This comment made me realize something. 

Most libraries would do `#[packable(tag_ty = "u8", error = "Infallible")]` because the quotes give a clear delimitation of each argument. But we avoid the quotes at all, which means we don't know when an argument ends because they can have arbitrary syntax. 

So if we have  `#[packable(tag_ty = u8, other_arg = (foo, trait T), error = Infallible)]`, how do we parse that? 
- If we were to skip tokens until we reach a `,` we will end up at `trait T`.
- If we try to parse `(foo, trait T)` we would fail because it is not a valid term in Rust.

Maybe we should just use quotes for arguments.